packages/office-ui-fabric-react/src/components/Keytip/examples/Keytips.Button.Example.tsx

```diff
@@ -21,78 +25,81 @@ export class KeytipsButtonExample extends React.Component<{}, IKeytipsButtonExam
-          text="Compound Button"
-          secondaryText={'With a Keytip'}
-          onClick={() => {
-            console.log('Compound Button');
-          }}
-        />
-        <DefaultButton
-          className="u-marginRight"
-          keytipProps={keytipMap.ButtonWithMenu}
-          text="Button with Menu"
-          onClick={() => {
-            console.log('Button with Menu');
-          }}
-          menuProps={{
-            items: [
-              {
-                key: 'buttonMenuItem1',
-                text: 'Menu Item 1',
-                keytipProps: keytipMap.ButtonMenuItem1
-              },
-              {
-                key: 'buttonMenuItem2',
-                text: 'Menu Item 2',
-                keytipProps: keytipMap.ButtonMenuItem2
-              }
-            ]
-          }}
-        />
-        <DefaultButton
-          className="u-marginRight"
-          keytipProps={keytipMap.SplitButton}
-          text="Split Button"
-          split={true}
-          onClick={() => {
-            console.log('Split Button');
-          }}
-          menuProps={{
-            items: [
-              {
-                key: 'splitButtonMenuButton1',
-                text: 'Split Button Menu Item 1',
-                keytipProps: keytipMap.SplitButtonMenuItem1
-              },
-              {
-                key: 'splitButtonMenuButton2',
-                text: 'Split Button Menu Item 2',
-                keytipProps: keytipMap.SplitButtonMenuItem2
-              }
-            ]
-          }}
-        />
-        <DefaultButton text="I do not have a keytip" />
+        <Stack horizontal gap={20}>
+          <DefaultButton
+            className="u-marginRight"
+            keytipProps={keytipMap.Button}
+            text="Button"
+            onClick={() => {
+              console.log('Button');
+            }}
+          />
+          <CompoundButton
+            styles={{ root: marginBottom }}
+            keytipProps={keytipMap.CompoundButton}
+            text="Compound Button"
+            secondaryText={'With a Keytip'}
+            onClick={() => {
+              console.log('Compound Button');
+            }}
+          />
+          <DefaultButton
+            keytipProps={keytipMap.ButtonWithMenu}
+            text="Button with Menu"
+            onClick={() => {
+              console.log('Button with Menu');
+            }}
+            menuProps={{
+              items: [
+                {
+                  key: 'buttonMenuItem1',
+                  text: 'Menu Item 1',
+                  keytipProps: keytipMap.ButtonMenuItem1
+                },
+                {
+                  key: 'buttonMenuItem2',
+                  text: 'Menu Item 2',
+                  keytipProps: keytipMap.ButtonMenuItem2
+                }
+              ]
+            }}
+          />
+          <DefaultButton
+            styles={{ splitButtonContainer: { height: 32 } }}
```

Comment:
And the other actual change (3) is giving the split button a height--for some reason after changing the other styles, it was filling the height of the container rather than just being the same height as its main button.